bin/checkver.ps1

```diff
@@ -285,7 +285,11 @@ while ($in_progress -gt 0) {
             continue
         }
 
-        if ($url) {
+        if ($script) {
+            $source = 'the output of script'
+            $page = Invoke-Command ([scriptblock]::Create($script -join "`r`n"))
+        } elseif ($url) {
+            $source = $url
```

Comment:
The `$page` parameter can be utilized within the `script`. Please restore the original sequence and seek an alternative method to rectify the output.

Reply:
Sure. I'll fix it later.

Reply:
@niheaven Updated.